### DISCUSSION TITLE
Is there a method to automatically reject a user request based on certain criteria being met? iE: IMDb <2.5

### DISCUSSION THREAD

**Discussion by zobe123** [sct/overseerr#3774] [Feature Requests]
I would welcome the implementation of an automatic system that is able to reject user requests if certain pre-defined criteria are met. For instance, one such criterion could involve the rating of a requested content falling below a specified threshold, such as an IMDb rating below 2.5.

**Comment by itschasa** [comment]
Not currently, you can make a feature request for this on the issues tab.
Alternatively, you can enable manual approvals for requests, and check them yourself.
